docs/schedule/examples/index.de.md

```diff
@@ -0,0 +1,21 @@
+---
+search:
+  exclude: true
+---
+
+# GTFS Schedule
+
+Zum besseren Verständnis der GTFS und zur Erstellung von Tools, die GTFS lesen und schreiben können, sehen Sie sich die folgenden Beispiele für GTFS an:
+
+- Die [Seite GTFS](/schedule/example-feed) bietet einen einfachen GTFS, der als Modell verwendet werden kann, um zu verstehen, wie ein GTFS zusammengesetzt ist.
+- Beispiele nach Anwendungsfall:
+  - [Routen, Haltestellen und Fahrten](routes-stops-trips)
+  - [Kontinuierliche Haltestellen](continuous-stops)
+  - [Fahrpreise v1](fares-v1)
+  - [Fahrpreise v2](fares-v2)
+  - [Frequenzen](frequencies)
+  - [Fahrwege](pathways)
+  - [Übersetzungen](translations)
+  - [Informationen zur Einspeisung](feed-info)
+  - [Datensatz-Attribute](attributions)
```

Comment:
I'll only make this comment once but applies to all languages for this page - I am unsure which of these links should be translated or not. (ex: Fares v1 and v2 getting translated here)